### DISCUSSION TITLE
A few ambient sounds under CC0

### DISCUSSION THREAD

**Discussion by garsipal** [redeclipse/base#1347] [Maps and Content Showcase]
I found some sounds I made buried in my hard drive, these could be used as ambience for Red Eclipse maps.
The folder contains 11 .ogg files, some of them have been tested in maps:

- water_drops.ogg - tested in map **wet**; please replace **cavedrip.ogg** as it is an ancient sound we have been hearing in every single Cube game since 2004;
- alien_world.ogg - tested in map **venus**, works great and adds to the alienness of the map;
- city_15.ogg - tested in maps **ubik** and **echo**, works decently, however this sound is a bit too long, I apologize for that;
- city_16.ogg - tested in map **edge**, works quite well and adds to the atmosphere;
- eerie_ambience.ogg - tested in map **bloodlust**, works very nice, it has an eerie feeling that complements the map's atmosphere;
- war_distant.ogg - tested in map **conquest**, it is a decent sound but it needs the right context that this map does not provide.

I'll see what I can come up with next, I might make some more ambient sounds when I get some inspiration.
[ambience.zip](https://github.com/redeclipse/base/files/8667355/ambience.zip)

**Comment by garsipal** [comment]
More ambience (13 more sounds). Some of these sounds were adjusted specifically for Red Eclipse.

The hightlights of this second pack are:
- alien_crickets.ogg - for more alien maps;
- city_15.ogg - shorter and lighter version
- crickets_loop.ogg , city_birds.ogg - decent quality recording;
- electro.ogg - should be useful as a replacement for the original **electro.ogg** ambience sound;
- water_drops_fast.ogg , water_drops_reverb.ogg - intended as more suitable alternatives to **cavedrip.ogg** (these might need volume adjustments);


[ambience.zip](https://github.com/redeclipse/base/files/8681382/ambience.zip)

**Comment by qreeves** [comment]
Thanks for that 👍

**Comment by garsipal** [comment]
Proper license file.
[license.txt](https://github.com/redeclipse/base/files/9264719/license.txt)
